ci-operator/step-registry/ipi/conf/gcp/oidc-creds-deprovision/ipi-conf-gcp-oidc-creds-deprovision-commands.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+infra_name=${NAMESPACE}-${JOB_NAME_HASH}
+export GCP_SHARED_CREDENTIALS_FILE=${CLUSTER_PROFILE_DIR}/gce.json
+export GOOGLE_APPLICATION_CREDENTIALS="${GCP_SHARED_CREDENTIALS_FILE}"
+PROJECT="$(< ${CLUSTER_PROFILE_DIR}/openshift_gcp_project)"
+
+# extract ccoctl from the release image
+CCO_IMAGE=$(oc adm release info --image-for='cloud-credential-operator' "$RELEASE_IMAGE_LATEST")
+cd "/tmp"
+oc image extract "$CCO_IMAGE" --file="/usr/bin/ccoctl"
+chmod 555 "/tmp/ccoctl"
```

Comment:
nit, the above `extract` will unpack into the current directory, which happens to be `/tmp`.  In the unlikely event that we pick a different working directory later, we'd avoid breaking by usnig `chmod 555 ccoctl` here.  And also `./ccoctl gcp delete ...` below, etc.